src/main/java/mate/academy/dbconnection/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.dbconnection;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";
+    private static final Properties DB_PROPERTIES;
```

Comment:
It is not secure to store database credentials in code. It's better to use environment variables or a configuration file that is not included in the version control.